google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -543,17 +553,32 @@ def update
         #   file.download "path/to/downloaded/gzipped.txt",
         #                 skip_decompress: true
         #
-        def download path = nil, verify: :md5, encryption_key: nil,
+        # @example Partially download.
+        #
+        #   require "google/cloud/storage"
+        #
+        #   storage = Google::Cloud::Storage.new
+        #   bucket = storage.bucket "my-bucket"
+        #   file = bucket.file "path/to/my-file.ext"
+        #
+        #   downloaded = file.download range: "bytes=3-6"
+        #   downloaded.rewind
```

Comment:
Looking for context: Why did you perform a `rewind` after download in this example?

Reply:
`download` doesn’t rewind the StringIO it returns after writing. The caller must rewind for a subsequent read to return the downloaded file contents.